apps/condo/domains/common/components/Table/Filters.tsx

```diff
@@ -86,17 +87,30 @@ export const getSelectFilterDropdown = (selectProps: ComponentProps<typeof Selec
                 showClearButton={selectedKeys && selectedKeys.length > 0}
                 style={containerStyles}
             >
-                <Select
-                    showArrow
-                    style={DROPDOWN_SELECT_STYLE}
-                    value={selectedKeys}
-                    optionFilterProp={'label'}
-                    onChange={(e) => {
-                        setSelectedKeys(e)
-                        confirm({ closeDropdown: false })
-                    }}
-                    {...selectProps}
-                />
+                {selectProps.mode === 'tags' ?
+                    <SelectTags
```

Comment:
I don't see any differecnce between this two <Select ...> props

Reply:
SelectTags and Select :) 
But I think attributes can be declared once and sent to both components